trafficserver_exporter/collector.py

```diff
@@ -708,6 +730,34 @@ def _parse_volume_metrics(self, data, volume):
         )
         yield metric
 
+        metric = Metric(
+            "trafficserver_cache_direntries_total",
```

Comment:
For these two I'd like to drop the `_total` suffix since they're gauges.

Reply:
Done.